src/ReflectionFileNamespace.php

```diff
@@ -357,6 +362,20 @@ private function findConstants()
             }
         }
 
+        if ($withDefinedConstants) {
+            foreach ($this->namespaceNode->stmts as $namespaceLevelNode) {
+                if ($namespaceLevelNode instanceof FuncCall && (string)$namespaceLevelNode->name === 'define') {
```

Comment:
What PHP code should like to have `FuncCall` class `name` property to be instance of `Expr` instead of `Name`?